### PR TITLE
OBSDOCS-1225: Logging 5.9.5 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Logging Z-Stream Release Notes - 5.9.5
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1225

Fix Version: 4.13, 4.14, 4.15, 4.16

Doc Preview: https://80344--ocpdocs-pr.netlify.app/openshift-dedicated/latest/observability/logging/logging_release_notes/logging-5-9-release-notes.html#logging-release-notes-5-9-5_logging-5-9-release-notes

SME Review: @xperimental 
QE Review: @kabirbhartiRH 
Peer Review: @ShaunaDiaz

### CODE REVIEW

**Comment by kabirbhartiRH** [comment]
@shreyasiddhartha Could you also add [LOG-5895](https://issues.redhat.com/browse/LOG-5895. Sorry to add it late, I could only reproduce it today.

File: modules/logging-release-notes-5-9-5.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-5_{context}"]
+= Logging 5.9.5
+This release includes link:https://access.redhat.com/errata/RHBA-2024:5122[OpenShift Logging Bug Fix Release 5.9.5]
+
+[id="logging-release-notes-5-9-5-bug-fixes"]
```
Comment (ShaunaDiaz) [inline modules/logging-release-notes-5-9-5.adoc:7]: If you want to link to bug fixes anywhere, you can add a content tag here.

Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-9-5.adoc:7]: @ShaunaDiaz I will discuss this with the team if that's required. We might start doing it with the next set of z-streams. :)

**Pull request merged**